SwiftWisdom/Core/StandardLibrary/Numbers/UnsignedInteger+Extensions.swift

```diff
@@ -24,14 +16,38 @@ public func random(inRange range: CountableClosedRange<Int>) -> Int {
     return random
 }
 
-// TODO: write tests for this extension
 extension UnsignedInteger {
 
     public init(ip_data: Data) {
         let hexInt = ip_data.ip_hexInt ?? 0
         self.init(ip_safely: hexInt)
     }
-
+    
+    public init<T : SignedInteger>(ip_safely value: T) {
```

Comment:
They are used to avoid overflow/underflow when converting between types. I'm potentially using one as part of the ip_random() function for my other PR (in progress.)